lte/gateway/c/core/oai/tasks/s1ap/s1ap_mme.hpp

```diff
@@ -22,9 +22,10 @@
   \email: lionel.gauthier@eurecom.fr
 */
 
-#ifndef FILE_S1AP_MME_SEEN
-#define FILE_S1AP_MME_SEEN
-
+#pragma once
+#ifdef __cplusplus
```

Comment:
There should be a TODO here too to remove this after all the files are migrated to C++.

Reply:
As suggested, the occurrences  of "ifdef __cplusplus" is at multiple places. Plan to remove on final migration to c++.
Created issue, https://github.com/magma/magma/issues/12785